fzf.el

```diff
@@ -244,6 +244,25 @@ If nil, fzf prompts have the same history in all modes."
   :safe #'booleanp
   :group 'fzf)
 
+(defcustom fzf/position-bottom-of-frame nil
+  "Set the position of the fzf to always be at the bottom of the frame."
+  :type 'boolean
+  :safe #'booleanp
+  :group 'fzf)
+
+(defun fzf/position-bottom-of-frame-watcher (symbol newval operation where)
+  "Watcher function for `fzf/position-bottom-of-frame`.
```

Comment:
Use: `fzf/position-bottom-of-frame' (replacing the trailing backtick with a single quote character).

Please use docstring markup compliant with the [elisp manual section D.6](https://www.gnu.org/software/emacs/manual/html_node/elisp/Documentation-Tips.html).  You may also want to refer to [Xahlee Elisp:Docstring Markup](http://xahlee.info/emacs/emacs/inline_doc.html) page.

Reply:
Fixed in 18ede3b292324cbae7f4decdbd07543abaf1980f